### DISCUSSION TITLE
2011/03/17/%E8%B1%86%E8%B1%86%E7%99%BE%E5%A4%A9%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#319] [Announcements]
# 2011/03/17/%E8%B1%86%E8%B1%86%E7%99%BE%E5%A4%A9%E4%BA%86

时间过得真快，豆豆一转眼就出生百天了。

https://blog.qizhen.xyz/2011/03/17/%E8%B1%86%E8%B1%86%E7%99%BE%E5%A4%A9%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

勺子
2011年03月19日 12:46 编辑
跟你挺像的呀（如果那个人是你）！

回复
